projects/cps-mdoc-viewer/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cps-mdoc-viewer",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^17.3.0",
+    "@angular/core": "^17.3.0",
+    "cps-ui-kit": "^17.26.0",
+    "gray-matter": "^4.0.3",
+    "js-yaml": "^4.1.0",
+    "marked": "^12.0.2",
+    "ngx-markdown": "^17.2.1"
+  },
+  "dependencies": {
```

Comment:
To be added
 
 "repository": {
    "type": "git",
    "url": "https://github.com/AbsaOSS/cps-mdoc-viewer"
  },
  "homepage": "https://github.com/AbsaOSS/cps-mdoc-viewer",
  "license": "Apache License 2.0",

Reply:
Added